### PR TITLE
Worked on and tested abstract class, its methods. Created objects, using polymorphism

### PR DESCRIPTION
**Pull request by ValentineCherniy**

### CODE REVIEW

**Comment by ValentineCherniy** [comment]
Please, check my 8th homework

File: src/main/java/homework_nr_8/Main.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        Appliance app_1 = new WashingMachine("LG", 2000, 7.5);
+        Appliance app_2 = new Microwave("Samsung", 1200, true);
+        Appliance app_3 = new AirConditioner("Daikin", 1500, 12000, true);
+
+        Appliance[] appliances = new Appliance[]{
+                app_1, app_2, app_3
+        };
+        for (int i=0; i !=appliances.length; i++){
+            System.out.println(appliances[i].displayInfo());
+            if (appliances[i] instanceof WashingMachine) {
```
Comment (gh3rtzz) [inline src/main/java/homework_nr_8/Main.java:14]: Use the better syntax of instanceof `element instanceOf Type castedBufferElement`

File: src/main/java/homework_nr_8/Main.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                System.out.println("This is a washing mashine with a load capacity " + ((WashingMachine) app_1).loadCapacity + " kg");
+            }
+            if (appliances[i] instanceof Microwave) {
```
Comment (gh3rtzz) [inline src/main/java/homework_nr_8/Main.java:17]: here too

File: src/main/java/homework_nr_8/Main.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+                if (((Microwave) app_2).hasGrill){
+                    System.out.println("This is a microwave with a grill");
+                } else {
+                    System.out.println("This is a microwave without a grill");
+                };
+            }
+            if (appliances[i] instanceof AirConditioner) {
```
Comment (gh3rtzz) [inline src/main/java/homework_nr_8/Main.java:24]: and here too

**Comment by ValentineCherniy** [comment]
done. Sorry, I've deleted you comments, bcz of low skills (don't know how to procees them in IDE)

**Review by gh3rtzz — APPROVED** [review]
approved

**Pull request merged**